core/common/src/main/java/alluxio/wire/TieredIdentity.java

```diff
@@ -169,6 +170,41 @@ public String getTierName() {
     public String getValue() {
       return mValue;
     }
+
+    /**
+     * @param otherTier wire type locality tier to compare to
+     * @return true if the wire type locality tier is local in comparison to the given tier
+     */
+    public boolean checkLocality(LocalityTier otherTier) {
```

Comment:
do you think it makes sense to make this a part of the `LocalityTier#equals` implementation? Then the check would be `tier.equals(otherTier)`

Reply:
Semantically speaking they are not exactly equal more like equivalent, nevertheless seems to simplify the code quite nicely, I will change this as well. Thanks!